core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/functions.scala

```diff
@@ -270,6 +270,7 @@ class CallMapper {
     case CallFunction("TO_CSV", args) => StructsToCsv(args.head, args(1))
     case CallFunction("TO_DATE", args) => ParseToDate(args.head, args(1))
     case CallFunction("TO_JSON", args) => StructsToJson(args.head, args(1))
+    case CallFunction("TO_NUMBER", args) => ToNumber(args.head, args(1))
```

Comment:
we don't have `ToNumber` in Spark. Make the conversion in optimizer rule

Reply:
https://docs.databricks.com/en/sql/language-manual/functions/to_number.html